app/models/text_message_access_token.rb

```diff
@@ -14,7 +16,7 @@ class TextMessageAccessToken < ApplicationRecord
   validates_presence_of :token
   validates :sms_phone_number, phone: true, format: { with: /\A\+1[0-9]{10}\z/ }
```

Comment:
we've got a scope in both "access_token" classes that's v similar. Wondering whether it makes sense for these classes to inherit from a parent class called AccessToken so that we can ensure similar behavior when things change?

Reply:
Obviously, that's out of scope for this PR but just spitballing. I've not had a chance to review this area of the code yet.

Reply:
I agree it'd be good! I didn't want to add too many refactors here.